src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorSupervisor.scala

```diff
@@ -176,46 +177,41 @@ class ClusterMonitorSupervisor(monitorConfig: MonitorConfig, dataprocConfig: Dat
     clusters <- dbRef.inTransaction {
       _.clusterQuery.getClustersReadyToAutoFreeze()
     }
-    pauseableClusters <- clusters.toList.filterA {
-      cluster =>
-        jupyterProxyDAO.isAllKernalsIdle(cluster.googleProject, cluster.clusterName).attempt.map {
-          attempted =>
-            attempted match {
-              case Left(t) =>
-                logger.error(s"Fail to get kernel status for ${cluster.googleProject}/${cluster.clusterName} due to $t")
-                true
-              case Right(isIdle) =>
-                if (!isIdle) {
-                  val idleLimit = Duration.ofNanos(autoFreezeConfig.maxKernelBusyLimit.toNanos) // convert from FiniteDuration to java Duration
-                  val maxKernelActiveTimeExceeded = cluster.auditInfo.kernelFoundBusyDate match {
-                    case Some(attemptedDate) => Duration.between(attemptedDate, Instant.now()).compareTo(idleLimit) == 1
-                    case None => {
-                      dbRef.inTransaction { dataAccess =>
-                        dataAccess.clusterQuery.updateKernelFoundBusyDate(cluster.id, Instant.now())
-                      }
-                      false // max kernel active time has not been exceeded
-                    }
-                  }
-                  if (maxKernelActiveTimeExceeded){
-                    Metrics.newRelic.incrementCounterIO("maxKernelExceededClusters").unsafeRunAsync(_ => ())
-                    logger.info(s"Auto pausing ${cluster.googleProject}/${cluster.clusterName} due to exceeded max kernel active time")
-                  } else {
-                    Metrics.newRelic.incrementCounterIO("activeKernelClusters").unsafeRunAsync(_ => ())
-                    logger.info(s"Not going to auto pause cluster ${cluster.googleProject}/${cluster.clusterName} due to active kernels")
-                  }
-                  maxKernelActiveTimeExceeded
-
-                } else isIdle
+    pauseableClusters <- clusters.toList.filterA[Future] { cluster =>
```

Comment:
There are no functional changes here -- just minor formatting cleanup, and fixes the red squiggles in Intellij.